src/main/java/org/apache/maven/plugin/resources/remote/ProcessRemoteResourcesMojo.java

```diff
@@ -847,8 +853,7 @@ protected boolean copyResourceIfExists( File file, String relFileName, VelocityC
                 }
                 else if ( resource.isFiltering() )
                 {
-
-                    MavenFileFilterRequest req = setupRequest( resource, source, file );
+                    MavenFileFilterRequest req = setupRequest( source, file, resource.isFiltering() );
```

Comment:
`req = setupRequest( source, file, true );`

Reply:
Thanks for the suggestion. This actually makes the third parameter 'true' for all invocations, so it can be removed. 4a1a5219ea29a754600fa1aa9cdcaca9f059fec2